views/subscribers/importExport/import.html

```diff
@@ -113,7 +113,7 @@ <h1 class="title">
 'pickListsDescription': __('Pick the list that you want to import these subscribers to.'),
 'select': _x('Select', ' Verb'),
 'createANewList': __('Create a new list'),
-'updateExistingSubscribers': __("Update existing subscribers' information"),
+'updateExistingSubscribers': __("Update existing subscribers' information, like first name, last name, etc."),
```

Comment:
@wxa When updating this translation, I think we should also change the regular apostrophe to the typographic one.  We do that for the new translations everywhere. It has many benefits as stated previously. (mainly because it looks better)

Also, why do you keep assigning Jan Lysy? I thought the new developers always start doing code reviews after a couple of weeks. Do I misremember that?

Reply:
>  change the regular apostrophe to the typographic one

Fixed. 
> We do that for the new translations everywhere

Could you please point me to a discussion or a document where this was set as a rule? I have only found it mentioned in a few comments to PRs I did not participate in.

> why do you keep assigning Jan Lysy? I thought the new developers always start doing code reviews after a couple of weeks. Do I misremember that?

Well, it's his third week already, so all good 😃 But seriously, I have asked him beforehand if he can review PRs and he said yes. And I did not mean to overwhelm him, the PRs are tiny, more to get him familiar with the process than anything else. Is something wrong with that?

Reply:
> Could you please point me to a discussion or a document where this was set as a rule?

That is not a rule, but that has been discussed a couple of times during various pull requests. Also, we discussed it with Quentin where we agreed that devs will do it but that has been a very long time ago and I don't remember if we did it on slack or Jira or maybe even in person.

>  Is something wrong with that?

no, there is nothing wrong with that, I am just surprised, that's it.